overrides/default-settings.gschema.override

```diff
@@ -136,18 +135,10 @@ active=false
 [org.gnome.settings-daemon.plugins.color]
 night-light-temperature=4500
 
-[org.gnome.settings-daemon.plugins.cursor]
-# Workaround upstream gnome-settings-daemon bug (https://bugzilla.gnome.org/show_bug.cgi?id=694758) as tracked by elementary (https://bugs.launchpad.net/elementaryos/+bug/1248747)
-active=false
-
 [org.gnome.settings-daemon.plugins.media-keys]
 screensaver='<Super>l'
 terminal='<Super>t'
 
-[org.gnome.settings-daemon.plugins.power]
-ambient-enabled=false
```

Comment:
I forget why this was disabled; I think because it seems dishonest to tell people it's on when it's unclear if it's actually supported? If we can actually detect if it's supported, that would be better. @danrabbit do you remember?

Reply:
@cassidyjames It's likely there's no ui for this and g-s-d just disables it when appropriate.

Reply:
@worldofpeace there is a UI; check _System Settings_ → _Power_ on a laptop, or the equivalent panel in GNOME. The problem may have also been that it's a poor experience by default; I think @danrabbit had made this call originally so I'm going to defer to him; I also do not have hardware with an ambient light sensor.

Reply:
I think at the time we disabled it, support was really bad. It could be better now. It's worth re-evaluating